apps/slackbot/Dockerfile

```
@@ -1,4 +1,4 @@
-FROM --platform=linux/amd64 python:3.12-slim@sha256:423ed6ab25b1921a477529254bfeeabf5855151dc2c3141699a1bfc852199fbf
+FROM --platform=linux/amd64 python:3.12.13-slim@sha256:423ed6ab25b1921a477529254bfeeabf5855151dc2c3141699a1bfc852199fbf
```

Hadolint (2.14.0)

[warning] Do not use --platform flag with FROM

(DL3029)